python/tvm/meta_schedule/space_generator/post_order_apply.py

```diff
@@ -29,8 +29,8 @@ class PostOrderApply(SpaceGenerator):
     rules to blocks in post-DFS order.
     """
 
-    def __init__(self):
+    def __init__(self, filter_fn=None):
```

Comment:
Nit, shall we unify the function name as `f_block_filter`?

Reply:
I actually think `filter_fn` is more clear than `f_block_filter` (using f for functions in python is unusual). I could change the c++ names if you want them to be consistent.

Reply:
Just to clarify, in previous cases we've been using f_func_name as PackedFunc argument name, for example in ApplyHistoryBest dispatch function. But I would be okay with using filter_fn here as long as it's consistent with c++ side.

Reply:
Ok that seems reasonable. I've updated all names to `f_block_filter` :).